keyboards/yandrstudio/zhou65/f401/matrix.c

```diff
@@ -0,0 +1,236 @@
+/* Copyright 2021 JasonRen(biu)
```

Comment:
Is there a reason for the custom matrix? 

And if so, could you convert this to matrix lite? 
https://docs.qmk.fm/#/custom_matrix?id=lite

Reply:
If the qmk default matrix_scan function is used, the pin A9 of stm32f4xx chip that using OTG as usb driver cannot be used as the matrix pin, which has been mentioned in [#7855](https://github.com/qmk/qmk_firmware/issues/7855) and [#11099](https://github.com/qmk/qmk_firmware/issues/11099). In my implementation, the output pin outputs a high state during matrix scanning, and the input pin reads the high state to trigger matrix instead of low state in qmk now.

Reply:
Except you dont use A9 on this keyboard?

Reply:
Yes, for STM32F4 series keyboards, if A9 is not used as a matrix pin, then the default configuration of QMK will work well.

Reply:
This keyboard does not use A9 pin. I have deleted the relevant files.